res/controllers/Ion Discover DJ.midi.xml

```diff
@@ -96,6 +87,15 @@
                 </options>
             </control>
             <control>
+                <group>[Channel1]</group>
+                <key>IonDiscoverDJ.jog_wheel</key>
+                <status>0xB0</status>
+                <midino>0x19</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
```

Comment:
All this diff seems to be showing is moving the `<control>` element for mapping the jog wheel further down in the file.